src/components/EditCollectiveForm.js

```diff
@@ -636,14 +636,16 @@ class EditCollectiveForm extends React.Component {
             >
               <FormattedMessage id="editCollective.menu.connectedAccounts" defaultMessage="Connected Accounts" />
             </MenuItem>
-            <MenuItem
-              selected={this.state.section === 'webhooks'}
-              route="editCollective"
-              params={{ slug: collective.slug, section: 'webhooks' }}
-              className="MenuItem webhooks"
-            >
-              <FormattedMessage id="editCollective.menu.webhooks" defaultMessage="Webhooks" />
-            </MenuItem>
+            {get(collective, 'settings.ENABLE_WEBHOOKS') && (
```

Comment:
We usually don't write collective settings like that. `enableWebhooks`?

While we're there, maybe we can also support a global feature flag to activate it for everyone. Will likely be useful in development or staging environments. In this case, `ENABLE_WEBHOOKS` is the usual spelling.

Reply:
@znarf Should this be in an environment variable?

Reply:
@eloyekunle yes, I think @Betree will take care of that.

Reply:
Okay.

Reply:
Fixed by enabling both `ENABLE_WEBHOOKS` in env and `enableWebhooks` in collective settings